container-chains/runtime-templates/simple/src/lib.rs

```diff
@@ -701,6 +705,7 @@ construct_runtime!(
         RootTesting: pallet_root_testing = 100,
         AsyncBacking: pallet_async_backing::{Pallet, Storage} = 110,
 
+        OffchainWorker: pallet_ocw_simple::{Pallet,Call,Event<T>} = 120,
```

Comment:
What would happen if a data preserver enables this? would it just throw an event?

It's a pity that we cannot enable this just for testing :(

Reply:
I would call it pallet_roo_ocw

Reply:
Perhaps one way in which we can improve this is by adding a extrinsic that enables the offchain worker (by having a storage item in the pallet OffchainWorkerTestEnabled.

Reply:
Good points... I will add the storage item and rename the pallet

Reply:
Pallet renamed in https://github.com/moondance-labs/tanssi/pull/723/commits/e7ff792fad52b899b70dfb355ee15387442125bb . Added storage variable and extrinsic to switch on/off the offchain worker in https://github.com/moondance-labs/tanssi/pull/723/commits/bbee0863271446d03998a3cb071381f190c0a39d . Disabled offchain worker activity by default in [d6afff8](https://github.com/moondance-labs/tanssi/pull/723/commits/d6afff86e4149c6667af05d99e207638ae7922c6) .

Reply:
Event emission using unsigned extrinic in [527989c](https://github.com/moondance-labs/tanssi/pull/723/commits/527989cdd1475f273200e0f6fa9fd9944ced680b)